glide.yaml

```diff
@@ -5,11 +5,15 @@ import:
   subpackages:
   - edwards25519
 - package: github.com/btcsuite/btclog
-- package: github.com/btcsuite/go-flags
+  version: 84c8d2346e9fc8c7b947e243b9c24e6df9fd206a
+- package: github.com/jessevdk/go-flags
+  version: ^1.3.0
 - package: github.com/btcsuite/go-socks
+  version: 4720035b7bfd2a9bb130b1c184f8bbe41b6f0d0f
```

Comment:
Specific commit versions should not be in the yaml.  That is what the lock file is for.